app/controllers/tenants_controller.rb

```diff
@@ -4,4 +4,31 @@ def index
 
   def show
   end
+
+  def new
+  end
+
+  def create
```

Comment:
Haha. So, maybe we should have a waffle ticket to refactor this? Almost like the Gilded Rose Kata...jk, not nearly that bad :smile: